lightning/src/ln/channel.rs

```diff
@@ -3386,6 +3418,29 @@ impl<SP: Deref> ChannelContext<SP> where SP::Target: SignerProvider {
 		(context.holder_selected_channel_reserve_satoshis, context.counterparty_selected_channel_reserve_satoshis)
 	}
 
+	/// Check that a balance value meets the channel reserve requirements or violates them (below reserve).
+	/// The channel value is an input, so that this can be used for checks with new planned channel value.
```

Comment:
Not sure if the reader would know that "new planned channel value" pertains to splicing. Could you clarify the comment a bit?